bindings/go/client/README.md

```diff
@@ -0,0 +1,54 @@
+
+High level Client for Motr
+==========================
+
+To run this sample, first start standalone motr by running command below:
+
+```
+> sudo su root
+> cd cortx-motr/motr/examples/
+>./setup_a_running_motr_system.sh
+```
+
+Go to directory: `cortx-motr/bindings/go`
+
+Modify `mio/mio.go` find the variable and adding value as below. 
+Because these variables are package private, that was no way to modify in `client.go`.
+
+```
+var verbose bool = true
+var threadsN int = 1
```

Comment:
These are not private variables. See how they are changed in mcp example.

Reply:
@andriytk Are you mean `mio/mio.go` ? I do see those variable value is set in `mio.go`  using command-line.

Because of how the way of code from `client` invoke code from `mio`, instead of get from command-line, it need a way to initial those variable in order to make it work.

I do think couple of way how to do it:

1. Make the variable capital case. ( so it is able public access outside of `mio` package )
2. Set a default value for those variable. ( but less flexible in code perspective, current implementation ) 
3. Make public setter for those variable. ( this look like better option, we could set those value at runtime ).

Any thought ? ( or I had misunderstanding you question )

Reply:
Yes, `mio/mio.go` automatically adds `-verbose` and `-threads` command-line options to manipulate these variables. `mcp/mcp.go` uses that. Why doesn't it work for you?

Reply:
Sorry, I don't understand this. Why does it work for mcp and doesn't work for your code?

Reply:
This is because `client` package do invoke method implement` io.Writer.Write()` from `mio.go`.

In `mio.go`:
```
// code implement io.Writer
func (mio *Mio) Write(p []byte) (n int, err error) {
    return mio.write(p, &mio.off)
}
```

```
func (mio *Mio) write(p []byte, off *int64) (n int, err error) {
    
    // some code omit here
    // alloc() is using threadsN
    if err = v.alloc(); err != nil {
        return 0, err
    }

```

```
func (v *iov) alloc() error {
    // threadsN may had no value, e.g value is not passing from command line,
    v.buf = make([]C.struct_m0_bufvec, threadsN)
```

Since this method is not call from command-line,  `threadsN`  default to zero and cause `client` code hang at runtime.

Reply:
Oh, I mean code in `client` package is indirectly calling io.Writer.Write() and eventually call `alloc()` and stuck there because `threadsN` suppose to be set to non zero before the `alloc()` invoke.

Reply:
`threadsN` default value is 1, as you can see from the code.

Reply:
Sorry to confuse, the original [mio.go](https://github.com/seagate/cortx-motr/blob/main/bindings/go/mio/mio.go#L132) is no set any value.

I was thinking maybe we could have a public method ( also for verbose variable ) to set the variable value ? so it could set any desired value.

Reply:
Ah.. ok, I had saw the code. 

Because the `client` code is not pass value through cmdline, I need to set the value some where. ( in this case, is in `mio.go`) 

Perhaps we could have a default value ? or public setter method ?

Reply:
@andriytk I remove the change for mio.go and let user modify the file instead if they want to run client.go